go.mod

```diff
@@ -2,49 +2,49 @@ module k8s.io/kops
 
 go 1.14
 
-// Version kubernetes-1.19.0-rc.4 => tag v0.19.0-rc.4
+// Version kubernetes-1.19.0 => tag v0.19.0
 
-replace k8s.io/api => k8s.io/api v0.19.0-rc.4
+replace k8s.io/api => k8s.io/api v0.19.0
 
-replace k8s.io/apimachinery => k8s.io/apimachinery v0.19.0-rc.4
+replace k8s.io/apimachinery => k8s.io/apimachinery v0.19.0
 
-replace k8s.io/client-go => k8s.io/client-go v0.19.0-rc.4
+replace k8s.io/client-go => k8s.io/client-go v0.19.0
 
-replace k8s.io/cloud-provider => k8s.io/cloud-provider v0.19.0-rc.4
+replace k8s.io/cloud-provider => k8s.io/cloud-provider v0.19.0
 
-replace k8s.io/kubectl => k8s.io/kubectl v0.19.0-rc.4
+replace k8s.io/kubectl => k8s.io/kubectl v0.19.0
 
-replace k8s.io/apiserver => k8s.io/apiserver v0.19.0-rc.4
+replace k8s.io/apiserver => k8s.io/apiserver v0.19.0
 
-replace k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.19.0-rc.4
+replace k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.19.0
 
-replace k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.19.0-rc.4
+replace k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.19.0
 
-replace k8s.io/kube-proxy => k8s.io/kube-proxy v0.19.0-rc.4
+replace k8s.io/kube-proxy => k8s.io/kube-proxy v0.19.0
 
-replace k8s.io/cri-api => k8s.io/cri-api v0.19.0-rc.4
+replace k8s.io/cri-api => k8s.io/cri-api v0.19.0
 
-replace k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.19.0-rc.4
+replace k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.19.0
 
-replace k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.19.0-rc.4
+replace k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.19.0
 
-replace k8s.io/component-base => k8s.io/component-base v0.19.0-rc.4
+replace k8s.io/component-base => k8s.io/component-base v0.19.0
 
-replace k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.19.0-rc.4
+replace k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.19.0
 
-replace k8s.io/metrics => k8s.io/metrics v0.19.0-rc.4
+replace k8s.io/metrics => k8s.io/metrics v0.19.0
 
-replace k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.19.0-rc.4
+replace k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.19.0
 
-replace k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.19.0-rc.4
+replace k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.19.0
 
-replace k8s.io/kubelet => k8s.io/kubelet v0.19.0-rc.4
+replace k8s.io/kubelet => k8s.io/kubelet v0.19.0
 
-replace k8s.io/cli-runtime => k8s.io/cli-runtime v0.19.0-rc.4
+replace k8s.io/cli-runtime => k8s.io/cli-runtime v0.19.0
 
-replace k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.19.0-rc.4
+replace k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.19.0
 
-replace k8s.io/code-generator => k8s.io/code-generator v0.19.0-rc.4
+replace k8s.io/code-generator => k8s.io/code-generator v0.19.0
 
 replace github.com/gophercloud/gophercloud => github.com/gophercloud/gophercloud v0.11.0
```

Comment:
Could you try remove this line? It shouldn't be necessary now that k8s doesn't directly depend on it. gomod should hopefully just use the version we specify and there should only be one line in go.sum.

Reply:
Not yet :)
```
go: k8s.io/cloud-provider-openstack@v1.18.0 requires
	k8s.io/kubernetes@v1.18.0 requires
	k8s.io/code-generator@v0.0.0: reading k8s.io/code-generator/go.mod at revision v0.0.0: unknown revision v0.0.0
```

Reply:
https://github.com/kubernetes/cloud-provider-openstack/pull/1151